Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM manageiq/ruby:latest
+
+# For pg gem required by manageiq-gems-pending
+RUN yum -y install postgresql-devel
```

Comment:
Can we consolidate the yum installs to one place, and do the yum-cleanup step as well?